_security-analytics/api-tools/alert-finding-api.md

```diff
@@ -83,7 +84,10 @@ State | Description
 `DELETED` | Someone deleted the detector or trigger associated with this alert while the alert was ongoing.
 
 ---
-## Acknowledge Alerts
+
+## Acknowledge alerts
+
+Gives an acknowledgement when an alert is triggered.
```

Comment:
"Provides" or "Sends" instead of "Gives"?

Reply:
```suggestion
Sends an acknowledgement when an alert is triggered.
```